.github/actions/sync/shared-config.rb

```diff
@@ -173,6 +173,14 @@ def git(*args)
   end
 end
 
+# Update Gemfile.lock if it exists, based on the Ruby version.
+if (target_directory_path/"Gemfile.lock").exist?
+  Dir.chdir target_directory_path do
+    puts "Running bundle install..."
+    system "bundle install --quiet >/dev/null"
```

Comment:
`bundle update --ruby` should be cheaper.

Could also maybe pass `--bundler` too to update the locked bundler version used.

Reply:
https://github.com/Homebrew/.github/pull/129